src/Console/Migrate.php

```diff
@@ -123,8 +199,31 @@ protected function resolveBuilders(): void
         $this->target = $this->resolveBuilder($this->target());
     }
 
+    protected function resolveOptions(): void
+    {
+        $this->tableNames =   $this->tableNames();
+        $this->excludeTables =   $this->excludeTables();
+
+        if ($this->tableNames[0] == '-' && $this->confirm('Get all table list from target connection(incase if source connection does not support it)?', false))
+            $this->getTablesFromTarget = true;
+
+        if ($this->confirm('Please choose option whether to truncate target table before transfer?', false))
+            $this->truncateTables = true;
+
+        if (($this->tableNames[0] == '-'
```

Comment:
1. Use strict comparison - `===`.
2. Move the comparison to a separate method.
3. The logical block is not wrapped in curly braces.